src/schema/nodes/bibliography_item.ts

```diff
@@ -58,97 +59,4 @@ export const bibliographyItem: NodeSpec = {
   },
   selectable: false,
   group: 'block',
-  parseDOM: [
```

Comment:
Do we not use parse dom for encoding?

Reply:
I don't believe so. I am able to import JATS, edit the content, and export JATS successfully.